src/main/java/org/gridsuite/study/server/NetworkModificationService.java

```diff
@@ -385,4 +385,52 @@ public Mono<Void> reorderModification(UUID groupUuid, UUID modificationUuid, UUI
             .bodyToMono(Void.class);
 
     }
+
+    public Flux<ModificationInfos> lineSplitWithVoltageLevel(UUID studyUuid, String lineSplitWithVoltageLevelAttributes,
+        UUID groupUuid, ModificationType modificationType, String variantId, UUID modificationUuid) {
+
+        Objects.requireNonNull(studyUuid);
+        Objects.requireNonNull(lineSplitWithVoltageLevelAttributes);
+
+        if (modificationUuid == null) {
```

Comment:
Split into 2 methods (create and update) ?

Reply:
Duplicate the code, just to mimic equipment creation hypotheses ?
I don't subscribe to this.